webpack/react_app/components/JobWizard/JobWizard.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Wizard } from '@patternfly/react-core';
+import { translate as __ } from 'foremanReact/common/I18n';
+
+export const JobWizard = () => {
+  const steps = [
```

Comment:
just a thought for the future - this steps array looks like a great candidate for adding a slot  :)

Reply:
Added a task for the tracker :) https://projects.theforeman.org/issues/31248